7.2/Dockerfile

```diff
@@ -34,6 +34,9 @@ ENV PATH=./vendor/bin:/composer/vendor/bin:/root/.yarn/bin:/usr/local/sbin:/usr/
 ENV COMPOSER_ALLOW_SUPERUSER=1
 RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
 
+# Install hirak/prestissimo package
+composer global require hirak/prestissimo
```

Comment:
This needs to be `RUN composer global require hirak/prestissimo`. Can you adjust that?

Reply:
Oh, how i missed that 🤦‍♂ . Fixed.